README.md

```diff
@@ -14,4 +14,4 @@ see https://hub.docker.com/r/reactnativecommunity/react-native-android/
 
 Name may change later, see https://github.com/react-native-community/docker-android/issues/7.
 
-Original version is split from react-native repo, see https://github.com/facebook/react-native/blob/master/ContainerShip/Dockerfile.android-base.
+Original version is split from react-native repo, see https://github.com/facebook/react-native/blob/9abe34ca78861a9445529d00a468176f115c803f/DockerTests.md .
```

Comment:
Make it point to previous `Dockerfile.android-base`

Reply:
In this commit https://github.com/facebook/react-native/commit/9351dd6c69cfc395b4ca40052c28491e92367289#diff-ea132857375789999a28a5a2b5e90179 this file was deleted